codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/generators/FluentClientDecorator.kt

```diff
@@ -555,13 +555,13 @@ class FluentClientGenerator(
 fun generateOperationShapeDocs(writer: RustWriter, symbolProvider: SymbolProvider, operation: OperationShape, model: Model): List<String> {
     val input = operation.inputShape(model)
     val fluentBuilderFullyQualifiedName = operation.fullyQualifiedFluentBuilder(symbolProvider)
-    return input.members().map { member ->
-        val builderInputDoc = member.asFluentBuilderInputDoc(symbolProvider)
-        val builderInputLink = "$fluentBuilderFullyQualifiedName::${symbolProvider.toMemberName(member)}"
-        val builderSetterDoc = member.asFluentBuilderSetterDoc(symbolProvider)
-        val builderSetterLink = "$fluentBuilderFullyQualifiedName::${member.setterName()}"
+    return input.members().map { memberShape ->
+        val builderInputDoc = memberShape.asFluentBuilderInputDoc(symbolProvider)
+        val builderInputLink = "$fluentBuilderFullyQualifiedName::${memberShape.memberName.toSnakeCase()}"
```

Comment:
The symbol provider should still be used to get the member name since that will change the member names in cases where it conflicts with reserved names, but feed it into the `docLink` function to remove the `r#`.